rules/DowngradePhp80/Rector/NullsafeMethodCall/DowngradeNullsafeToTernaryOperatorRector.php

```diff
@@ -29,8 +42,8 @@ public function getRuleDefinition(): RuleDefinition
 CODE_SAMPLE
 ,
                 <<<'CODE_SAMPLE'
-$dateAsString = $booking->getStartDate() ? $booking->getStartDate()->asDateTimeString() : null;
-$dateAsString = $booking->startDate ? $booking->startDate->dateTimeString : null;
+$dateAsString = ($_ = $booking->getStartDate()) ? $_->asDateTimeString() : null;
+$dateAsString = ($_ = $booking->startDate) ? $_->dateTimeString : null;
```

Comment:
I'm partial to leaving this as-is just so people don't rely on variable names (for some reason 🤔) but happy to change it to reflect what it actually transforms it into. Thoughts?

Reply:
the example need to have same behaviour with one of fixtures